src/index.ts

```diff
@@ -63,7 +63,9 @@ export async function reply(message: Message, options: string | ReplyMessageOpti
 
 function resolvePayload(target: MessageTarget, options: string | Options, extra?: Options | undefined): Promise<MessagePayload> {
 	options =
-		typeof options === 'string' ? { content: options, embeds: [], attachments: [] } : { content: null, embeds: [], attachments: [], ...options };
+		typeof options === 'string'
+			? { content: options, embeds: [], attachments: [], components: [], files: [], allowedMentions: undefined }
+			: { content: null, embeds: [], attachments: [], components: [], files: [], allowedMentions: undefined, ...options };
```

Comment:
`files` and `allowedMentions` are not necessary here, they don't spread across updates.